### PR TITLE
Update user guide for filter and link command

### PR DESCRIPTION
**Pull request by xyT-T**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -241,5 +278,6 @@ Action     | Format, Examples
 **Add Tag** | `add_tag t/TAG_NAME` <br> e.g. `add_tag t/vendor`
 **View Tags** | `view_tags`
 **Delete Tag** | `delete_tag t/tag_name` <br> e.g. `delete_tag t/vendor`
+**Filter** | `filter TAG_NAME` <br> e.g. `filter vendor`
```
Comment (rionshocker) [inline docs/UserGuide.md:281]: Missing the command for Link

**Review by rionshocker — CHANGES_REQUESTED** [review]
Apart from missing the link command in the command summary, the rest lgtm!

File: docs/UserGuide.md (hunk 1 above)
Comment (xyT-T) [reply docs/UserGuide.md]: Ok thanks! Have just updated it.

**Review by rionshocker — APPROVED** [review]
LGTM!

**Pull request merged**